src/pages/full-day.mdx

```diff
@@ -11,53 +12,74 @@ Join us at **#ContentDay**!
 - **When?** June 8, 2024 
 - **Where?** Institute of English Studies at Jagiellonian University, Kraków
 
-{/* 
-
 <br/>
 
-## Agenda
+{/* ## Agenda
 
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 <Tabs>
-  <TabItem
-    value="presentations"
-    label="Presentations"
-    default
-  >
+<TabItem
+    value="workshops"
+    label="Workshops"
+    default>
 
 Coming soon!
+</TabItem>
 
-  </TabItem>
-  <TabItem
-    value="workshops"
-    label="Workshops"
-  >
+<TabItem
+    value="presentations"
+    label="Presentations"
+    default>
 
 Coming soon!
 
   </TabItem>
-</Tabs> 
+</Tabs>  */}
 
-Coming soon!
-*/}
 
-<br/>
+## Sign up
 
-## Call for speakers and trainers
+Are you as excited as we are? Join us for workshops and presentations! 
 
-Is there a content-related topic that you're particularly passionate about? Are
-you ready to share your expertise with the community? That's great news! Propose
-your presentation or workshop with this form: https://forms.gle/aKPDg8S6PFsdRRsq8
+### Workshops
 
-We're waiting for your proposals until May 20.
+Our workshops run in parallel at 10AM-12PM, which means you can attend just one of them.
+> The number of spots is limited and on a first-come, first-serve basis.
 
-<br/>
+<details>
+  <summary>**Design (thinking) a doc site** with Paweł Kowaluk</summary>
+  <div>
+    **About the workshop** <br/>
+    Learn how to apply design thinking to creating a doc website. We will create a persona and use that to create the sitemap for our documentation site. <br/>
+    **About the trainer** <br/>
+    Paweł Kowaluk has been in the tech writing industry since 2008. He has worked as a tech writer, manager, project manager, consultant, salesman, instructor, and doc tools developer. Currently, he does technical writing for Guidewire Software and serves as a board member at ITCQF. Additionally, Paweł explores the technical side of content creation as a co-founder and host of the https://techwriterkoduje.pl podcast. <br/>
+    **What to bring**: For this workshop you will need a laptop.
```

Comment:
You don't need a laptop for this workshop

Reply:
Oops, descriptions got mixed up, sorry about that!